EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+from typing import Callable, Optional
+from time import time as _time
+import asyncio
+from ..exceptions import TaskFailedError
+
+_EPOCH = datetime.datetime(1970, 1, 1)
+
+
+class TimeParser:
+    @staticmethod
+    def get_time_since_epoch(
+        year: int, month: int, day: int, hour: int, minute: int, seconds: float
+    ) -> float:
+        date = datetime.datetime(year, month, day, hour, minute, seconds)
+
+        delta = date - _EPOCH
+        return delta.total_seconds()
+
+    @staticmethod
+    def total_seconds(
+        year: int = 0,
```

Comment:
i dont the the point in having `month` & `year` honestly.
Plus note that a month != 30 as it varies between [28, 31]. Year also will not take leap into account. 

So, if you want to make a system that would work every X of a month, i would rather have something like a daily check, or a crontab implementation (which could be really useful).

As for year, ngl, it pretty much a joke

Reply:
True.

Reply:
yeah, i wrote a comment abt it and i deleted it, it is a headache

Reply:
i will remove month and year then?

Reply:
Yes.